docs/community/governance.md

```diff
@@ -73,28 +73,82 @@ long experience with both the project and the larger ecosystem. When the core
 developer community (including the SC members) fails to reach such a consensus
 in a reasonable timeframe, the SC is the entity that resolves the issue.
 
-Members of the steering council also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
+Members of the SC also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
 and are ultimately responsible for managing the napari GitHub account, the [@napari_imaging](https://twitter.com/napari_imaging)
 twitter account, the [napari website](http://napari.org), and other similar napari owned resources.
 
-The steering council is currently fixed in size to three members. This number may be increased in
-the future, but will always be an odd number to ensure a simple majority vote outcome
-is always possible. The initial steering council of napari consists of
-
-- [Juan Nunez-Iglesias](https://github.com/jni)
-- [Loic Royer](https://github.com/royerloic)
-- [Nicholas Sofroniew](https://github.com/sofroniewn)
-
-The SC membership is revisited every January. SC members who do
-not actively engage with the SC duties are expected to resign. New members are
-added by nomination by a core developer. Nominees should have demonstrated
+The napari steering council (SC) will be no less than three members and no
+more than five members, with a strong preference for an odd number to ensure
+a simple majority vote outcome is always possible, and a preference for five
+members to ensure a diversity of voices. A majority of the SC will not be
```

Comment:
not particularly concerned but worth highlighting: the possibility of four SC members is now very real, the fact that this could cause problems around decision making is raised but no mechanisms are defined for how to deal with this issue if it arises

Reply:
I agree with @alisterburt 's concern, but I have faith in the SC to work it out. Perhaps some wording on what to do in the rare case that an agreement can't be made amongst the SC members would be helpful for the future.

Also, I'm curious about the limit of five members? I think five is a great number and makes sense, but what about seven? Is there a particular reason for the limit at five?

Reply:
> Perhaps some wording on what to do in the rare case that an agreement can't be made amongst the SC members would be helpful for the future.

I have added a line that if the vote is deadlocked it will be deferred until an odd number of members is present.

> Is there a particular reason for the limit at five?

It was thought to be a good limit with one IFP seat. The open forcefield project has 7 members with two IFP seats.